### ISSUE TITLE
Modelica.Math functions declared as "C" not "builtin"

### ISSUE THREAD

**Issue by modelica-trac-importer** [modelica/Modelica#371]
**Reported by PHarman on 24 May 2010 09:38 UTC**
Section 12.9 of the spec defines "builtin" as one of the external languages and gives the example:
```
package Modelica
    package Math
        function sin
            input Real x;
            output Real y;
            external "builtin";
        end sin;
    end Math;
end Modelica;
```

Section 3.7.1.2 lists sin, cos, tan, asin, acos, atan, atan2, sinh, cosh, tanh, exp, log and log10 as the builtin functions and states "They are also available as external built-in functions in the Modelica.Math library", yet in Modelica.Math these functions are declared as 'external "C"' not 'external "builtin"'.

----

Migrated-From: https://trac.modelica.org/Modelica/ticket/371

**Comment by modelica-trac-importer** [comment]
**Comment by dietmarw on 24 May 2010 16:19 UTC**
There doesn't exist a released version MSL3.2 (yet) ;)

**Comment by modelica-trac-importer** [comment]
**Comment by otter on 16 Jul 2010 09:52 UTC**
Fixed in 2e3866ee1421d2953f709b5d98ad0f977faf7503.
